Topic 1/src/point6/Offer.java

```diff
@@ -0,0 +1,32 @@
+package point6;
+
+
+import java.util.List;
+
+public class Offer extends AbstrsctProduct {
+	private List<Product> offers;
```

Comment:
[High] According exercise 6, the offer can include individual items or other offers.